docs/lang/python.md

```diff
@@ -0,0 +1,105 @@
+# Building a Python Environment
+
+This guide covers configuring Python environments in envd. If you’re new to envd please read our [Tutorial](../get-started) and [build configuration guides](../build-envd) first.
+
+Let's begin 🐍!
+
+## Specifying Python
+
+The default language in envd is Python, thus there is no need to specify language. Or you can use `base` function to specify.
+
+```python title=build.envd
+def build():
+    base(os="ubuntu20.04", language="python")
+```
+
+## Conda packages
+
+You can install conda packages with `install.conda_packages` function. The following example installs `numpy` and `scipy`:
+
+```python title=build.envd
+def build():
+    base(os="ubuntu20.04", language="python")
+    install.conda_packages(name = [
+        "numpy",
+        "scipy",
+    ])
+```
+
+## PyPI packages
+
+You can install Python packages from PyPI with `install.python_packages` function. The following example installs `scikit-learn` and `matplotlib`:
+
+```python title=build.envd
+def build():
+    base(os="ubuntu20.04", language="python")
+    install.python_packages(name = [
+        "scikit-learn",
+        "matplotlib",
+    ])
+```
+
+envd uses system-wide [pip](https://pip.pypa.io/) to install Python packages in the previous example.
```

Comment:
`system-wide` is ambiguous, which can also refer to the python delivered with the system, not the one we installed.

Reply:
The system does not have Python, thus we installed a python using apt install in the base image.

Thus we can consider the python we installed as system-wide python. I think.

Reply:
Is there still a python2 delivered with the system? Also I think apt install is just a temporary way since it has only one version?

Reply:
>  Also I think apt install is just a temporary way since it has only one version?

From the user's perspective, they can treat the python they used as the only python in the environment. From envd developer's perspective, it may not be the only python if conda is enabled. But I do not think we need to introduce it in the doc.

Reply:
SGTM